ckan/templates-midnight-blue/organization/snippets/helper.html

```diff
@@ -9,19 +9,27 @@
 
   #}
 
-
-<div class="module module-narrow module-shallow">
-  <h2 class="module-heading">
-    <i class="fa fa-info-circle"></i>
-    {{ _('What are Organizations?') }}
-  </h2>
-  <div class="module-content">
-    <p>
-      {% trans %}
-        CKAN Organizations are used to create, manage and publish collections
-        of datasets. Users can have different roles within an Organization,
-        depending on their level of authorisation to create, edit and publish.
-      {% endtrans %}
-    </p>
+<div class="module module-narrow module-shallow accordion">
```

Comment:
since this covers all the content in the sidebar what do we gain with the accordion here? Is this just for mobile users?

On desktop I feel that if we're closing the content of the sidebar the whole bar should collapse or something, instead of creating more white space.

Reply:
I think the accordions were initially to keep the sidebar consistent across the pages, but now we have two element designs we are using in the sidebar. We could go with the sidebar boxes in #8965 or I could go back to the designers get their rationale and see if it needs to be redesigned altogether.